src/main/java/com/exadel/discountwebapp/vendor/vo/VendorBaseVO.java

```diff
@@ -0,0 +1,31 @@
+package com.exadel.discountwebapp.vendor.vo;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.*;
+import java.io.Serializable;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+public class VendorBaseVO implements Serializable {
+    @NotBlank
+    @Size(min = 2, max = 50)
+    private String title;
+    @NotBlank
+    @Size(min = 2, max = 510)
+    private String description;
+    @NotBlank
+    @Size(min = 2, max = 510)
+    private String imageUrl;
+    @NotBlank
+    @Pattern(regexp = "(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|\"(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21\\x23-\\x5b\\x5d-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])*\")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21-\\x5a\\x53-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])+)\\])")
```

Comment:
Let's remove this complicated pattern for now) Or just add @Email annotation

Reply:
changed to @Email